src/xenia/kernel/xboxkrnl/xboxkrnl_rtl.cc

```diff
@@ -529,9 +529,10 @@ DECLARE_XBOXKRNL_EXPORT1(RtlTimeToTimeFields, kNone, kImplemented);
 dword_result_t RtlTimeFieldsToTime(pointer_t<X_TIME_FIELDS> time_fields_ptr,
                                    lpqword_t time_ptr) {
   if (time_fields_ptr->year < 1601 || time_fields_ptr->month < 1 ||
-      time_fields_ptr->month > 11 || time_fields_ptr->day < 1 ||
-      time_fields_ptr->hour > 23 || time_fields_ptr->minute > 59 ||
-      time_fields_ptr->second > 59 || time_fields_ptr->milliseconds > 999) {
+      time_fields_ptr->month > 12 || time_fields_ptr->day < 1 ||
+      time_fields_ptr->day > 31 || time_fields_ptr->hour > 23 ||
```

Comment:
Should we verify the number of days in the month according to the month and the year?

Reply:
No idea how it works under the hood, but we can clearly define that day of month never should be higher than 31. 
I guess we could run few tests on windows to see what will happen, but it still would be a blackbox test